build-test.xml

```diff
@@ -9766,7 +9767,13 @@ if (ppstate.equals("normal")) {
 	</target>
 
 	<target name="run-poshi-runner">
-		<poshi-execute task="runPoshi" />
+		<condition property="failonerror" value="false">
+			<not>
+				<equals arg1="${failonerror}" arg2="true" />
+			</not>
+		</condition>
```

Comment:
@lesliewong92 Are you sure you need this condition task? It seems redundant when poshi-execute has a default value of false for the failonerror attribute.

Reply:
If we don't have the condition, it'll set the attribute value to the String literal "${failonerror}", which will override the default attribute value.

Reply:
As an alternative, I can change it to an <if> statement, and have branched <poshi-execute> statements depending on if _failonerror_ is set or not to make it more explicit.